pkg/kubelet/dockershim/docker_sandbox.go

```diff
@@ -216,14 +216,34 @@ func (ds *dockerService) StopPodSandbox(podSandboxID string) error {
 // sandbox, they should be forcibly removed.
 func (ds *dockerService) RemovePodSandbox(podSandboxID string) error {
 	var errs []error
+	opts := dockertypes.ContainerListOptions{All: true}
```

Comment:
This doesn't cleanup the log symlink (see https://github.com/kubernetes/kubernetes/blob/master/pkg/kubelet/dockershim/docker_container.go#L296-L309).

And I'm not sure whether this will introduce more problem when contianer remove logic become more complex in the future.

Could we reuse `ListContainers` and `RemoveContainer` in dockershim?